BUILD.bazel

```diff
@@ -807,6 +809,17 @@ cc_test(
     ],
 )
 
+cc_library(
+    name = "gcs_test_util_lib",
```

Comment:
I think it is unnecessary to define this lib as it just include a simple `gcs_test_util.h`.